<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryTransfer</class>
 <widget class="QDialog" name="InventoryTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="date_Label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="from_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="transfer_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tranfert : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transfer_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_dateEdit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="fromName_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="release_PushButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>580</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Release</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="from_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>De : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="to_LineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toName_LineEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="to_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>À :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantity_LineEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="allocated_Label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Alloué : </string>
   </property>
  </widget>
  <widget class="ItemTable" name="tableWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>721</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="quantity_Label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité : </string>
   </property>
  </widget>
  <widget class="ItemWidget" name="itemWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="allocated_LineEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>180</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemTable</class>
   <extends>QWidget</extends>
   <header>PromotedClass/itemtable.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemWidget</class>
   <extends>QWidget</extends>
   <header>PromotedClass/itemwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>from_LineEdit</tabstop>
  <tabstop>to_LineEdit</tabstop>
  <tabstop>date_dateEdit</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>fromName_LineEdit</tabstop>
  <tabstop>release_PushButton</tabstop>
  <tabstop>transfer_LineEdit</tabstop>
  <tabstop>toName_LineEdit</tabstop>
  <tabstop>quantity_LineEdit</tabstop>
  <tabstop>allocated_LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
